fabric_generator/fabric_gen.py

```diff
@@ -15,6380 +15,2278 @@
+        self.writer.addComment("BEL ports (e.g., slices)", onNewLine=True)
+        repeatDeclaration = set()
+        for bel in tile.bels:
+            for i in bel.inputs + bel.outputs:
+                if f"{i}" not in repeatDeclaration:
+                    self.writer.addConnectionScalar(i)
+                    repeatDeclaration.add(f"{bel.prefix}{i}")
+
+        # Jump wires
+        self.writer.addComment("Jump wires", onNewLine=True)
+        for p in tile.portsInfo:
+            if p.wireDirection == Direction.JUMP:
+                if p.sourceName != "NULL" and p.destinationName != "NULL" and p.inOut == IO.OUTPUT:
+                    self.writer.addConnectionVector(p.name, f"{p.wireCount}-1")
+
+                for k in range(p.wireCount):
+                    allJumpWireList.append(f"{p.name}( {k} )")
+
+        # internal configuration data signal to daisy-chain all BELs (if any and in the order they are listed in the fabric.csv)
+        self.writer.addComment(
+            "internal configuration data signal to daisy-chain all BELs (if any and in the order they are listed in the fabric.csv)", onNewLine=True)
+
+        # the signal has to be number of BELs+2 bits wide (Bel_counter+1 downto 0)
+        # we chain switch matrices only to the configuration port, if they really contain configuration bits
+        # i.e. switch matrices have a config port which is indicated by "NumberOfConfigBits:0 is false"
+
+        # The following conditional as intended to only generate the config_data signal if really anything is actually configured
+        # however, we leave it and just use this signal as conf_data(0 downto 0) for simply touting through CONFin to CONFout
+        # maybe even useful if we want to add a buffer here
+
+        if tile.globalConfigBits > 0:
+            self.writer.addConnectionVector("ConfigBits", "NoConfigBits-1", 0)
+            self.writer.addConnectionVector(
+                "ConfigBits_N", "NoConfigBits-1", 0)
+
+            self.writer.addNewLine()
+            self.writer.addConnectionVector(
+                "FrameData_i", "FrameBitsPerRow-1", 0)
+            self.writer.addConnectionVector(
+                "FrameData_O_i", "FrameBitsPerRow-1", 0)
+            self.writer.addAssignScalar("FrameData_O_i", "FrameData_i")
+            self.writer.addNewLine()
+            for i in range(self.fabric.frameBitsPerRow):
+                self.writer.addInstantiation("my_buf",
+                                             f"data_inbuf_{i}",
+                                             ["A", "X"],
+                                             [f"frameData[{i}]", f"frameData_i[{i}]"])
+            for i in range(self.fabric.frameBitsPerRow):
+                self.writer.addInstantiation("my_buf",
+                                             f"data_outbuf_{i}",
+                                             ["A", "X"],
+                                             [f"frameData_O_i[{i}]", f"frameData_O[{i}]"])
```

Comment:
verilog is case-sensitive, so this needs to be `FrameData_O_i`, `FrameData_O`, etc to be consistent with elsewhere